frappe/utils/oauth.py

```diff
@@ -166,22 +164,12 @@ def get_info_via_oauth(provider, code, decoder=None, id_token=False):
 
 
 def login_oauth_user(
-	data=None, provider=None, state=None, email_id=None, key=None, generate_login_token=False
+	data: dict | str,
+	*,
+	provider: str | None = None,
+	state: dict | str,
+	generate_login_token: bool = False,
```

Comment:
reverted this.

saw some usages hence thought might be better to not break it in v14

https://sourcegraph.com/search?q=context:global+login_oauth_user%28&patternType=standard&sm=1